infra/core/monitor/monitoring.bicep

```diff
@@ -1,7 +1,7 @@
 metadata description = 'Creates an Application Insights instance and a Log Analytics workspace.'
 param logAnalyticsName string
 param applicationInsightsName string
-param applicationInsightsDashboardName string
+param applicationInsightsDashboardName string = ''
```

Comment:
@jongio I had to make this change to core so that it didnt force me to specify a dashboard name here. This seems somewhat redundant with includeDashboard, it could be gated on just the name being provided. What do you think? I know you want all changes propagated to azure-dev.